/home/bahaduri/VIPER/lamorel/lamorel/src/lamorel_launcher/launch.py:15: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path='', config_name='')
/home/bahaduri/.conda/envs/viper2/lib/python3.9/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
Error executing job with overrides: ['rl_script_args.path=/home/bahaduri/VIPER/experiments/Train_PPO.py', 'rl_script_args.output_dir=.', 'lamorel_args.accelerate_args.machine_rank=0', 'lamorel_args.llm_args.model_path=meta-llama/Llama-3.2-1B-Instruct', 'lamorel_args.llm_args.model_type=causal', 'rl_script_args.seed=3', 'rl_script_args.number_envs=1', 'rl_script_args.task=[1]', 'rl_script_args.output_dir=/home/bahaduri/VIPER/outputs/ppo_llm_wIntrinsic_pick', 'rl_script_args.intrinsic_reward=True', 'rl_script_args.loading_path=/home/bahaduri/VIPER/outputs/ppo_llm_wIntrinsic_pick/epochs_81-82', 'lamorel_args.config_alfred=/home/bahaduri/VIPER/alfworld/configs/base_config.yaml', 'wandb_args.run=Examine_in_light', 'lamorel_args.llm_args.vlm_model_path=microsoft/Florence-2-base-ft', 'wandb_args.mode=offline', 'lamorel_args.distributed_setup_args.n_llm_processes=1', 'rl_script_args.transitions_buffer_len=5', 'rl_script_args.epochs=500', 'rl_script_args.gradient_batch_size=2', 'rl_script_args.name_environment=AlfredThorEnv', 'rl_script_args.startepochs=83']
Traceback (most recent call last):
  File "/home/bahaduri/VIPER/lamorel/lamorel/src/lamorel_launcher/launch.py", line 46, in main
    launch_command(accelerate_args)
  File "/home/bahaduri/.conda/envs/viper2/lib/python3.9/site-packages/accelerate/commands/launch.py", line 1204, in launch_command
    multi_gpu_launcher(args)
  File "/home/bahaduri/.conda/envs/viper2/lib/python3.9/site-packages/accelerate/commands/launch.py", line 803, in multi_gpu_launcher
    current_env = prepare_multi_gpu_env(args)
  File "/home/bahaduri/.conda/envs/viper2/lib/python3.9/site-packages/accelerate/utils/launch.py", line 223, in prepare_multi_gpu_env
    raise ConnectionError(
ConnectionError: Tried to launch distributed communication on port `12347`, but another process is utilizing it. Please specify a different port (such as using the `--main_process_port` flag or specifying a different `main_process_port` in your config file) and rerun your script. To automatically use the next open port (on a single node), you can set this to `0`.

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
